db/migrate/20151222103721_migrate_provider_attributes_to_endpoints.rb

```diff
@@ -0,0 +1,41 @@
+class MigrateProviderAttributesToEndpoints < ActiveRecord::Migration
+  class Provider < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class Endpoint < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Migrating Provider attributes to Endpoints") do
+      ExtManagementSystem.all.each do |ems|
+        next if ems.provider_id.nil?
+        provider = Provider.where(:id => ems.provider_id).first
+        endpoint = Endpoint.where(
+          :resource_type => "ExtManagementSystem",
+          :resource_id   => ems.id).first_or_create
+
+        endpoint.update_attributes!(:verify_ssl => provider.verify_ssl)
+      end
+    end
+  end
+
+  def down
+    say_with_time("Migrating Endpoints to Provider attributes") do
+      endpoints = Endpoint.where(
+        :role          => "default",
+        :resource_type => "ExtManagementSystem",
+      )
+
+      endpoints.each do |endpoint|
+        next if endpoint.verify_ssl.nil?
+        ems = ExtManagementSystem.where(:id => endpoint.resource_id).first
+        ems.provider.update_attributes!(
```

Comment:
@juliancheal, @jrafanie Found the spec failure.  This should update the Provider and not the EMS record.  In fact, this should fail with some attribute not found exception, right?  Since `ExtManagementSystem` doesn't actually have a `verify_ssl` attribute.